client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -69,8 +71,21 @@ public String toString() {
                 '}';
     }
 
-    static Request ping() {
-        return new Request("HEAD", "/", Collections.emptyMap(), null);
+    // For developers please add your new methods in the alphabetical order
```

Comment:
@javanna It doesn't matter, that's not a convention and it doesn't make sense to add a comment for that. I just find it easier to find and read class methods when they are sorted, specially when it's a suite of operations like this. But if I'm the only one to care about this, let's add new method wherever you want.